backend/frontend/urls.py

```diff
@@ -39,6 +39,11 @@
         views.action_start_card,
         name="action_start_card",
     ),
+    path(
+        "cards/<int:card_id>/details",
+        views.link_card_details_page,
+        name="link_card_details_page",
```

Comment:
It's very likely that we'll use the same view for all project cards. So a better name would be "project_details_page".

If it is a link project then the template should show link stuff, and if it is a repo project, it should show repo stuff. We can handle this in the template or in the view.